util/BUILD

```diff
@@ -15,6 +16,17 @@ genrule(
     stamp = 1,  # this provides volatile-status.txt
 )
 
+py_binary(
+    name = "otbn_build",
+    srcs = ["otbn_build.py"],
+    imports = ["../hw/ip/otbn/util/"],
+    deps = [
+        requirement("pyelftools"),
+        "//hw/ip/otbn/util:otbn_as",
+        "//hw/ip/otbn/util:otbn_ld",
```

Comment:
An alternative to source dependencies is to add these as `data` dependencies and shell out to them as programs.  I don't have a particular preference - either method is ok by me.